R/app_ui.R

```diff
@@ -61,6 +61,47 @@ app_ui <- function() {
         )
       ),
 
+      shiny::navbarMenu(
+        "Figurer/tabeller",
+        shiny::tabPanel(
+          "Antall ferdige/uferdige skjema",
+          shiny::sidebarLayout(
+            shiny::sidebarPanel(
+              shiny::dateRangeInput("dato_antall_skjema",
+                label = "Dato:",
+                start = "2014-01-01",
+                end = lubridate::today(),
+                min = "2014-01-01",
+                max = lubridate::today(),
+                language = "no",
+                weekstart = 1,
+                separator = " til "
+              ),
+              shiny::actionButton("alle_datoer_knapp", "Alt"),
+              shiny::actionButton("innevarende_aar_knapp", "Inneværende år"),
+              shiny::actionButton("tretti_dager_knapp", "Siste 30 dager"),
+              shiny::actionButton("syv_dager_knapp", "Siste 7 dager"),
+              shiny::checkboxGroupInput("alderkat",
+                label = "Alder:",
```

Comment:
Vi burde kanskje ha en regel for hvordan vi gjør det med InputId og label. Skal de defineres eksplisitt?

Reply:
Tenkte du på om me skal bruka namn på argumenta? Regelen i fylgje kodestilen vår er vel at argument som står på eigne linjer i alle fall skal stå med namn. Men `label` kunne gjerne vore brukt i linje 83 òg, sjølv om den står på éi linje.